scripts/flashida/write_method.py

```
@@ -0,0 +1,105 @@
+    {'key': 'CycleTime', 'name': 'CycleTime', 'value': 180},
+    {'key': 'CVValues', 'name': 'CVValues', 'value': [-10.0, -30.0, -40.0, -50.0, -60.0]},
+]
+
+def get_params():
+    if len(sys.argv) > 1:
+        with open(sys.argv[1], "r") as f:
+            return json.load(f)
+    else:
+        return {}
+
+if __name__ == "__main__":
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)